packages/create-llama/templates/components/use-cases/typescript/deep_research/src/app/workflow.ts

```diff
@@ -178,7 +180,7 @@ export function getWorkflow(index: VectorStoreIndex | LlamaCloudIndex) {
       }),
     );
 
-    const retrievedNodes = await retriever.retrieve(userInput);
+    const retrievedNodes = await retriever.retrieve(userInput as QueryType);
```

Comment:
```suggestion
    const retrievedNodes = await retriever.retrieve({ query: userInput });
```

see
`export type QueryBundle = {
  query: MessageContent;
  customEmbeddings?: string[];
  embeddings?: number[];
};

export type QueryType = string | QueryBundle;
`